docs/quickstart.md

```diff
@@ -0,0 +1,108 @@
+# Goose in 5 minutes
+
+## Quickstart guide
+
+Goose is a developer agent that supercharges your software development by automating an array of coding tasks directly within your terminal or IDE. You can get it setup and running on your devices in only a few minutes.
+
+### Installation
+
+To install Goose, use `pipx`. First ensure [pipx][pipx] is installed:
+
+``` sh
+brew install pipx
+pipx ensurepath
+```
+
+Then install Goose:
+
+```sh
+pipx install goose-ai
+```
+
+### Running Goose
+
+#### Start a session
+
+From your terminal, navigate to the directory you'd like to start from and run:
+
+```sh
+goose session start
+```
+
+#### Set up a provider
+Goose works on top of LLMs and by default, it uses `openai` as the LLM provider but you can customize it as needed. You'll be prompted to set an [OPENAI_API_KEY][openai-key] if you haven't set one previously.
+
+>[!TIP]
+> **Billing:**
+>
+> You will need to add credits to your Open AI accounts to be able to successfully make requests.
+>
+
+
+#### Make Goose do the work for you
+You will see the Goose prompt `G❯`:
+
+```
+G❯ type your instructions here exactly as you would tell a developer.
+```
+
+Now you are interacting with Goose in conversational sessions - think of it like you're giving directions to a junior developer. The default toolkit allows Goose to take actions through shell commands and file edits. You can interrupt Goose with `CTRL+D` or `ESC+Enter` at any time to help redirect its efforts.
+
+#### Exit the session
+
+If you are looking to exit, use `CTRL+D`, although Goose should help you figure that out if you forget.
+
+#### Resume a session
+
+When you exit a session, it will save the history in `~/.config/goose/sessions` directory and you can resume it later on:
+
+``` sh
+goose session resume
+```
```

Comment:
a good place to link the sessions management guide so they can know more actually about resuming a session and naming it..

maybe we be specific and say that goose session resume will resume your **last** saved session..but if you want to resume a different session check out the session management guide [link to the one ebony wrote]

Reply:
Yeah definitely but the guide hasn't been merged yet #246. Will add the message in the meantime.